src/instrumentations/xhr.ts

```diff
@@ -753,7 +807,24 @@ export function instrumentXHR(config: XhrConfig) {
                 }
               }
             } else if (this.readyState === XMLHttpRequest.DONE) {
-              endSpan(EventNames.EVENT_READY_STATE_CHANGE, this);
+              if (config.networkBodyCapture && this.responseType === 'blob') {
+                new Response(this.response)
+                  .text()
+                  .then((text) => {
+                    currentSpan.setAttribute('http.response.body', text);
+                  })
+                  .finally(() => {
+                    endSpan(EventNames.EVENT_READY_STATE_CHANGE, this);
```

Comment:
just double check. the span would never end before this ?

Reply:
it shouldn't